src/dotnet/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,3 @@
+using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("dotnet.Tests")]
```

Comment:
Hm, wasn't sure, trying to run it locally now. I thought you only needed that if the target assembly was strong name signed and sn said this one wasn't. I'm probably wrong though.